### PR TITLE
Improve styling

### LINKED ISSUE #38: Improve the look and feel of the app
The current style looks a bit sad.

image

Perhaps it needs a bit of makeup, like this one

image

### PR DESCRIPTION
**Pull request by quangkhoa**
Resolve #38

### CODE REVIEW

File: src/components/BodySchema/BodySchema.scss [hunk 1]
```diff
@@ -8,11 +8,14 @@
 }
 
 .body-schema--even {
-  background-color: $grey;
+  //background-color: rgb(40, 50, 65);
```
Comment (brendo) [inline src/components/BodySchema/BodySchema.scss]: Remove the commented out colours.

File: src/components/Property/Property.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ export default class Property extends PureComponent {
           <PropertyConstraints constraints={constraints} type={type} isRequired={isRequired} />
           {enumValues && this.renderEnumValues(enumValues)}
           {defaultValue !== undefined && this.renderDefaultValue(defaultValue)}
-          {description && <Description description={description} />}
+          {description && <div><Description isInline description={description} /></div>}
```
Comment (brendo) [inline src/components/Property/Property.js:64]: Curious about the `<div>` wrapper here, the Description component already provides a wrapper (with classes `description` and `description-inline`). What benefit does the second wrapper give?

Comment (quangkhoa) [reply src/components/Property/Property.js:64]: Without the `div`, the description on property will be rendered inline, i.e. on the same line.

If I remove `isInLine`, then the result will be wrapped in a `p` tag, which breaks the tree css. The current tree css is not desirable, but I'd like to deal with that at a slightly later stage.

Comment (brendo) [reply src/components/Property/Property.js:64]: Ok, leave as is 👍

File: src/components/BodySchema/BodySchema.scss (hunk 1 above)
Comment (quangkhoa) [reply src/components/BodySchema/BodySchema.scss]: removed

**Pull request merged**